Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.15.2-alpine3.12 as builder
+COPY . /src
+WORKDIR /src/cmd
+RUN apk add --no-cache git ca-certificates && \
+  go build -v .
+
+FROM alpine:3.12.0
+COPY --from=builder /src/cmd/cmd /bin/godolint
```

Comment:
There is no `/src/cmd/cmd`, so isn't it `/src/cmd` ?

Reply:
`/src/cmd` is the directory, `/src/cmd/cmd` is the output of `go build`

Reply:
This was a misunderstanding. As you pointed out 👍